modules/tracking/src/augmented_unscented_kalman.cpp

```diff
@@ -0,0 +1,358 @@
+#include "opencv2/tracking/kalman_filters.hpp"
```

Comment:
1. put the OpenCV license
2. put `#include "precomp.hpp"` before all other include's.